compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/assign_tiles.mlir

```diff
@@ -509,3 +509,215 @@ module attributes {hal.executable.target = #executable_target_amdaie_xclbin_fb}
     return
   }
 }
+
+// -----
```

Comment:
Could you add the same two tests for L2->L3 as well?

Reply:
I've added L2 -> L3 within the same and have marked the Copy section with comments in input case.